index.html

```diff
@@ -47,6 +48,15 @@ <h1>
         <iframe src="https://player.vimeo.com/video/438989702?autoplay=1&loop=1&muted=1" width="5120" height="2880" frameborder="0" allow="autoplay; fullscreen" allowfullscreen>
         </iframe>
       </div>
+      <div class='showreel-description'>
+        All work animated, illustrated and filmed by Beth Archibald, except for some background textures by 
+        <a href='http://www.drawingroom.nyc/store' class='textures-link' target='_blank'>The Drawing Room</a> and 
+        <a href='https://blog.spoongraphics.co.uk/resources' class='textures-link' target='_blank'>Spoon Graphics</a> 
+        <br>
+        <br>
+        <br>
+        Check out a video outlining my design process <a href='https://vimeo.com/439318214' class='vimeo-link' target='_blank'>HERE</a>
+      </div>
```

Comment:
Using what I mentioned below, you don't even have to give these two <a> tags classes